lib/seattleflu/id3c/cli/command/etl/clinical.py

```diff
@@ -463,15 +463,19 @@ def encounter_details(document: dict) -> dict:
     if "CovidScreen" in document:
         details["responses"]["CovidScreen"] = covid_screen(document.get("CovidScreen"))
 
-    for k in ["CovidShot1", "CovidShot2"]:
+    for k in ["CovidShot1", "CovidShot2", "CovidShot3"]:
         if k in document:
             details["responses"][k] = covid_shot(document[k])
 
     if "CovidShotManufacturer" in document:
         details["responses"]["CovidShotManufacturer"] = covid_shot_maunufacturer(document.get("CovidShotManufacturer"))
 
+    for k in ["CovidShot1Manu", "CovidShot2Manu", "CovidShot3Manu"]:
+        if document.get(k):
+            details["responses"][k] = [document[k]]
```

Comment:
Could use the `covid_shot_manufacturer` function here to validate and standardize incoming values.